src/lib/routes/admin-api/user-admin.ts

```diff
@@ -451,48 +529,66 @@ export default class UserAdminController extends Controller {
             );
         }
 
-        const responseData: UserSchema = {
+        const responseData: CreateUserResponseSchema = {
             ...serializeDates(createdUser),
             inviteLink: inviteLink || this.unleashUrl,
             emailSent,
-            rootRole,
+            rootRole: normalizedRootRole,
         };
 
         this.openApiService.respondWithValidation(
-            201,
+            200,
```

Comment:
Why did we change this from a 201 to a 200?

Reply:
I think one of our validators was complaining that 201 should have a Location header

Reply:
Yeah, that sounds about right. You should add one: `respondWithValidation` takes an optional last parameter called `headers`. You can add it in there, pointing to the newly created user.

Reply:
Ok I found the location header setting pattern in other parts of the code. Our schema was lying before with 200 and the code was returning 201. Let's stick to 201 in both